### ISSUE TITLE
Uncaught TypeError: Cannot read property '5' of undefined

### ISSUE THREAD

**Issue by tsukasa696** [joehoyle/atom-autocomplete-wordpress-hooks#27]
[Enter steps to reproduce:]

1. just open and some of the plugins fail i update/restart all the ones need it, but this one just fail in a middle of codding
2. ...

**Atom**: 1.19.0 x64
**Electron**: 1.6.9
**OS**: Unknown Windows version
**Thrown From**: [autocomplete-wordpress-hooks](https://github.com/joehoyle/atom-autocomplete-wordpress-hooks) package 0.2.2

### Stack Trace

Uncaught TypeError: Cannot read property '5' of undefined

```
At C:\Users\Tsukasa\.atom\packages\autocomplete-wordpress-hooks\lib\provider.js:111

TypeError: Cannot read property '5' of undefined
    at Object.isInFilter (/packages/autocomplete-wordpress-hooks/lib/provider.js:111:41)
    at Object.getSuggestions (/packages/autocomplete-wordpress-hooks/lib/provider.js:37:13)
    at ~/AppData/Local/atom/app-1.19.0/resources/app/node_modules/autocomplete-plus/lib/autocomplete-manager.js:337:60
    at Array.forEach (native)
    at AutocompleteManager.getSuggestionsFromProviders (~/AppData/Local/atom/app-1.19.0/resources/app/node_modules/autocomplete-plus/lib/autocomplete-manager.js:313:23)
    at AutocompleteManager.findSuggestions (~/AppData/Local/atom/app-1.19.0/resources/app/node_modules/autocomplete-plus/lib/autocomplete-manager.js:302:25)
```

### Commands

```
     -9:39.8.0 core:paste (input.hidden-input)
  6x -9:33.8.0 core:backspace (input.hidden-input)
     -9:30.9.0 intentions:highlight (input.hidden-input)
     -9:30.5.0 core:save (input.hidden-input)
  2x -9:11.6.0 intentions:highlight (input.hidden-input)
     -9:09.7.0 core:save (input.hidden-input)
     -5:13.8.0 tabs:close-other-tabs (div.title.icon.gear-icon)
     -0:19 emmet:insert-formatted-line-break-only (input.hidden-input)
     -0:19 editor:newline (input.hidden-input)
     -0:18.9.0 intentions:highlight (input.hidden-input)
     -0:18.7.0 core:paste (input.hidden-input)
 14x -0:17.8.0 core:move-up (input.hidden-input)
  3x -0:15.6.0 core:move-down (input.hidden-input)
     -0:14.9.0 core:move-up (input.hidden-input)
     -0:14.5.0 emmet:insert-formatted-line-break-only (input.hidden-input)
     -0:14.5.0 editor:newline (input.hidden-input)
```

### Non-Core Packages

```
aligner-php 1.1.1 
atom-autocomplete-php 0.25.6 
atom-beautify 0.30.4 
atom-runner 2.7.1 
autocomplete-php 0.3.7 
autocomplete-wordpress-hooks 0.2.2 
blade-snippets 0.2.0 
browser-plus 0.0.98 
busy-signal 1.4.3 
color-picker 2.2.5 
emmet 2.4.3 
emmet-snippets-compatibility 1.0.6 
file-icons 2.1.10 
filesize 3.0.0 
goto-definition 1.3.4 
highlight-line 0.12.0 
highlight-selected 0.13.1 
intentions 1.1.5 
language-blade 0.34.0 
laravel 0.7.1 
laravel-snippets 2.1.0 
linter 2.2.0 
linter-php 1.3.2 
linter-sass-lint 1.7.4 
linter-ui-default 1.6.4 
livereload 0.4.4 
livestyle-atom 0.2.9 
minimap 4.29.6 
minimap-pigments 0.2.2 
php-cs-fixer 4.1.0 
pigments 0.40.2 
wordpress-api 1.2.2 
wordpress-dictionary 4.4.0 
```

**Comment by ghost** [comment]
Same problem for me

**Comment by joehoyle** [comment]
Fixed in v0.2.3
